doc/md/guides/orms/gorm.md

```diff
@@ -290,7 +290,46 @@ Observe a new migration file is generated:
 -- Modify "pets" table
 ALTER TABLE `pets` ADD COLUMN `nickname` longtext NULL;
 ```
+#### View
+This is a unique API for the Atlas GORM Provider that allows you to define database views in the form of GORM models and automatically generates migration files for you.
+> The view feature is only available for logged-in users; run `atlas login` if you haven't already.
 
+To define a Go struct as a database `VIEW`, implement the `ViewDef` method as follows:
+```go
```

Comment:
Views are essentially saved queries, so to define them we must define a query. 

The `gormschema` package provide two styles for defining a view's base query:
* BuildStmt - explain what it is and when to use
* CreateStmt - explain what it is and when to use.
Then give a short example for each.